### DISCUSSION TITLE
Facing issues with test server

### DISCUSSION THREAD

**Discussion by MikiPAUL** [saran-surya/email_auth_node#47] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/79696324/199819681-42a02b5e-2bb4-49e2-858b-7db37638b97c.png)
I tried to use test server but getting this error, how to resolve this issue?

**Comment by saran-surya** [comment, marked as answer]
Test server is running fine, Kindly check if you are behind a proxy, and if its whitelisted.
https://github.com/saran-surya/email_auth_node/blob/main/README.md#test-server-working-status--
